src/components/Utils/Validate.js

```diff
@@ -4,8 +4,10 @@ const Required = (value) => {
   return 'Required!';
 };
 
-const RequiredInput = () => {
-  return false;
+const RequiredRefNumber = (value, initialValues) => {
```

Comment:
`RequiredRefNumber` quite hard to understand, maybe it's better to add comment like
`// field is required only if `vendor_detail.ref_number` isn't empty